src/plus/integrations/integrationService.ts

```diff
@@ -126,6 +126,7 @@ export class IntegrationService implements Disposable {
 	}
 
 	private onDidChangeSubscription(e: SubscriptionChangeEvent) {
+		// TODO is this correct?
```

Comment:
This ensures that when we log out of our GK account, any connected cloud integrations are disconnected.

Reply:
Ah -- Can you update that comment then please?